### ISSUE TITLE
Who counts as a collaborator?  

### ISSUE THREAD

**Issue by Gibble1** [github/game-off-2012#14]
I'm thinking about entering as an individual.  It was noted that open source media is acceptable, and the creator of such media would of course not be a collaborator, but suppose I used already produced audio with the permission of the artist, would that make the artist a collaborator?  

I'm assuming, although I wanted to ask this as well, that any artist creating new materials for the project would have to be considered as a collaborator, promoting my status from an individual to a team.

**Comment by leereilly** [comment]
That's a great question. If I understand you correctly, you're concerned that if you use artists' materials with permission that'd make it a team submission. I'm going to say that'll still be considered an individual entry. It'd be great to give the artist credit in the README file though if you think that's appropriate.

There'll likely be cases where team members / collaborators don't have any commits on the repository, but will have been instrumental in creating the game. I'll definitely contact the winners + runners up to confirm before announcing the winners.

**TL;DR:** I think you're fine with forking to your individual account :+1: 

Cheers,
  Lee :beers:
